packages/frontend/src/hooks/useQueryResults.tsx

```diff
@@ -83,7 +83,7 @@ const getQueryResults = async ({
     const timezoneFixQuery = query && {
         ...query,
         filters: convertDateFilters(query.filters),
-        timezone: dayjs.tz.guess(),
+        timezone: query.timezone || TimeZone.UTC,
```

Comment:
I think if not specified, this should be undefined, and let the backend handle it properly (without forcing to UTC)

Reply:
Not sure if we want to include a `default` timezone in the picker that makes this undefined. 


I'm saying this because I'm not sure if `SET timezone UTC` is the same as `not setting the timezone` in the warehouse.

Reply:
Note this will not be captured by the feature flag, so it will affect users as soon as we release this.

Reply:
Updated